### PR TITLE
mgr/dashboard: fixed password generation in Auth controller

### PR DESCRIPTION
**Pull request by rjfd**
Fixes: http://tracker.ceph.com/issues/23404

Signed-off-by: Ricardo Dias <rdias@suse.com>

### CODE REVIEW

File: src/pybind/mgr/dashboard/requirements.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 astroid==1.6.1
 attrs==17.4.0
 backports.functools-lru-cache==1.4
+bcrypt==3.1.4
```
Comment (sebastian-philipp) [inline src/pybind/mgr/dashboard/requirements.txt]: This seems to be a completely different package. Does this change have any affect on the spec and control files?

Comment (rjfd) [reply src/pybind/mgr/dashboard/requirements.txt]: no, the problem was actually the opposite. The bcrypt package installed in the several distros using the package manager was the `bcrypt` package, and not the `python-bcrypt` package. This change corrects the difference.

**Comment by rjfd** [comment]
QA tests were successful: http://pulpito.ceph.com/rdias-2018-03-22_16:43:51-rados:mgr-wip-rdias-testing-distro-basic-smithi/

**Comment by rjfd** [comment]
jenkins retest this please

**Pull request merged**